css/style.css

```diff
@@ -72,4 +73,14 @@ li {
 }
 span {
     color: black;
-}
\ No newline at end of file
+}
+@media screen and (max-width: 768px){
+    .header {font-size: 18px;
+        flex-direction: column;}
+    .introduction {font-size: 20px;}
+    .skills {font-size: 20px;}
```

Comment:
FYI: it's a good practice to use rem instead of px for responsible (adaptive) layout.  As it a relative value, It's convinient to define different font-size for different devices and all the sizes are adapted automatically depending on the used device.

Reply:
ty for comment, try this next time